src/assets/css/basic.css

```diff
@@ -137,3 +137,19 @@ body {
 .highzindex {
     z-index: 10000 !important;
 }
+
+.no-margin {
+    margin: 0;
+}
+
+.no-bottom-margin {
+    margin-bottom: 0;
+}
+
+.no-top-margin {
+    margin-top: 0;
+}
+
+.CodeMirror-vscrollbar {
+    display: none !important;
+}
```

Comment:
Maybe you can move this to MarkdownEditor.vue of scoped css

Reply:
It's not that simple, the MDE is based on native HTML. So it's rendered when you open the web in your browser. If you move this into vue's scoped css area, vue won't know what `CodeMirror-vscrollbar` is and ignore it, so it won't be rendered and replace the MDE's original CSS. Whereas you put it into `basic.css`, it won't be complied. It will be rendered when your browser runs `main.js` and then render `basic.css` so the MDE css can be replaced.
I've tried to place these three line in scoped area, but it failed to render, so I moved it here.
I've no idea why CodeMirror have these kind of strange codes. It's high enough, but there is always a scrollbar displays. Maybe the only way to fix this is to change MDE's source code, but you know, I don't want to... So this is it, I think placing these three lines in `basic.css` won't matter.

Reply:
By the way, I have no idea why CodeMirror wrote all this junk, I have to replace their css using IMPORTANT, why!! WHY!!

Reply:
Got it.
But, maybe there are some better choice